damus/Views/SearchResultsView.swift

```diff
@@ -51,7 +53,35 @@ struct InnerSearchResults: View {
     func HashtagSearch(_ ht: String) -> some View {
         let search_model = SearchModel(state: damus_state, search: .filter_hashtag([ht]))
         return NavigationLink(value: Route.Search(search: search_model)) {
-            Text("Search hashtag: #\(ht)", comment: "Navigation link to search hashtag.")
+            HStack {
+                Image("search")
+                Text("#\(ht)", comment: "Navigation link to search hashtag.")
+            }
+            .padding(.horizontal, 15)
+            .padding(.vertical, 5)
+            .background(DamusColors.neutral1)
+            .cornerRadius(20)
+            .overlay(
+                RoundedRectangle(cornerRadius: 20)
+                    .stroke(DamusColors.neutral3, lineWidth: 1)
+            )
+        }
+    }
+    
+    func TextSearch(_ txt: String) -> some View {
+        return NavigationLink(value: Route.NDBSearch(results: $results)) {
+            HStack {
+                Image("search")
+                Text("Notes", comment: "Navigation link to search text.")
+            }
+            .padding(.horizontal, 15)
+            .padding(.vertical, 5)
+            .background(DamusColors.neutral1)
+            .cornerRadius(20)
+            .overlay(
+                RoundedRectangle(cornerRadius: 20)
+                    .stroke(DamusColors.neutral3, lineWidth: 1)
+            )
```

Comment:
We should define this gray button style as a reusable view, to reduce code duplication between `TextSearch` and `HashtagSearch`